### PR TITLE
Custom Domains documentation updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated custom domains doc with GCP load balancer-specific settings.

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Custom domains update](https://www.notion.so/rudderstacks/Custom-Domains-doc-update-07b88a86c96541239583ed9edb970044)

### CODE REVIEW

File: docs/user-guides/how-to-guides/custom-domains.mdx [hunk 1]
```diff
@@ -259,6 +259,36 @@ You can choose your subdomain or use a wildcard domain <code class="inline-code"
...
+```markup
+<script src="https://subdomain.yourdomain.com/v1.1/rudder-analytics.min.js"></script>
+```
+
+The correct custom domain URL to access any device-mode SDK (for example, Google Analytics) should be in the following format: 
+
+```markup
+<script src="https://subdomain.yourdomain.com/v1.1/js-integrations/GA.min.js"></script>
+```
+
```
Comment (MoumitaM) [inline docs/user-guides/how-to-guides/custom-domains.mdx:291]: Add [this](https://www.rudderstack.com/docs/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/version-migration-guide/#case-3-self-hosting-rudderstacks-cdn) link here. 
`https://subdomain.yourdomain.com/v1.1/js-integrations` will basically be used as the `destSDKBaseURL` in load option.

Comment (psrikanth88) [reply docs/user-guides/how-to-guides/custom-domains.mdx:291]: @MoumitaM  @ameypv-rudder Should we update the first section `Setup for serving the SDK` to include code snippet to set the right `destSDKBaseURL` when using custom domains. Can include the snippet here again in FAQ just to reinforce

**Review by psrikanth88 — CHANGES_REQUESTED** [review]

File: docs/user-guides/how-to-guides/custom-domains.mdx (hunk 1 above)
Comment (MoumitaM) [reply docs/user-guides/how-to-guides/custom-domains.mdx:291]: @psrikanth88, Yes we should.

Comment (ishachopra0401) [reply docs/user-guides/how-to-guides/custom-domains.mdx:305]: @MoumitaM @psrikanth88 The requested changes have been addressed.

File: docs/user-guides/how-to-guides/custom-domains.mdx [hunk 2]
```diff
@@ -147,6 +147,16 @@ After:
 <script src="https://subdomain.yourdomain.com/v1.1/rudder-analytics.min.js"></script>
 ```
 
+To <Link to="/stream-sources/rudderstack-sdk-integration-guides/rudderstack-javascript-sdk/version-migration-guide/#case-3-self-hosting-rudderstacks-cdn">load a device-mode SDK</Link> (for example, Google Analytics), you can use the following URL: 
+
+```html
+<script>
+rudderanalytics.load(WRITE_KEY, DATA_PLANE_URL, {
+    destSDKBaseURL: "https://subdomain.yourdomain.com/v1.1/js-integrations/GA.min.js"
```
Comment (psrikanth88) [inline docs/user-guides/how-to-guides/custom-domains.mdx:155]: the destSDKBaseURL should not contain the `js-integrations/GA.min.js` part ?

**Review by MoumitaM — APPROVED** [review]

**Review by ishachopra0401 — APPROVED** [review]

**Review by psrikanth88 — APPROVED** [review]

**Pull request merged**